content/applications/websites/website/web_design/themes.rst

```diff
@@ -0,0 +1,47 @@
+==============
+Website themes
+==============
+
+Odoo Themes provides a set of colors, fonts, and layouts to shape your website's style and configure
+various customization options. When setting up your site using the Odoo website configurator,
+selecting a theme that aligns with your desired aesthetic is imperative. If you wish to modify the
+theme later, navigate to the website builder, click the 'Edit' button, and access the 'Theme' tab.
+
+Diverse sections are at your disposal, empowering you to tailor different aspects of your site to
+meet your requirements.
+
+- :guilabel:`Colors`: The Website Builder relies on palettes composed of five colors. By default,
+  Odoo pre-selected colors based on your company logo. Defining those colors in your theme ensures
+  you stay consistent. Click :guilabel:`Color Presets` to configure your favorite color
+  combinations.
+
+- :guilabel:`Website`: From this section, you can :guilabel:`Switch Theme`,
+  :guilabel:`Add a Language`, select the :guilabel:`Page Layout`, and customize the
+  :guilabel:`Background`.
+
+  .. seealso::
+     :doc:`Translations <../configuration/translate>`
+
+- :guilabel:`Paragraph`: This section is related to the format of your text <p>.
+
+- :guilabel:`Headings`: This section is related to the format of your headings.
+
+- :guilabel:`Button`: Two types of buttons exist in Odoo: the :guilabel:`Primary Style` and the
+  :guilabel:`Secondary Style`. You can update their style from this section.
```

Comment:
It would be nice to know what are the rules of when the primary and secondary button types are used on the pages (if we know)